frontend/taipy-gui/src/components/Taipy/Input.tsx

```diff
@@ -138,6 +138,20 @@ const Input = (props: TaipyInputProps) => {
         [changeDelay, dispatch, updateVarName, module, onChange, propagate]
     );
 
+    const handleBlur = useCallback(
+        (e: React.FocusEvent<HTMLInputElement>) => {
+            const val = e.target.value;
+            if (changeDelay > 0 && delayCall.current > 0) {
+                clearTimeout(delayCall.current);
+                delayCall.current = -1;
+                dispatch(createSendUpdateAction(updateVarName, val, module, onChange, propagate));
```

Comment:
this should always be called 
So put outside the if/else block

Reply:
But I still don't see any tests ;-)
Are you still working on this @kart2004  ?

Reply:
Tests were added.